src/main/java/emu/grasscutter/server/packet/recv/HandlerSetPlayerBornDataReq.java

```diff
@@ -28,10 +28,13 @@ public void handle(GameSession session, byte[] header, byte[] payload) throws Ex
         // Sanity checks
         int avatarId = req.getAvatarId();
         int startingSkillDepot;
+        int elementlessSkillDepot;
```

Comment:
This should not need hardcoding, since the skillDepotId for both travelers is already managing this

Reply:
Handling for non questing could probably also be improved here by setting the skill depot for starting via element, instead of hardcoding the ids, though thats not something you touched in this pr.

Reply:
Yeah originally I had this here since I noticed skill depot was being set to pyro, but it was resolved in another file - this was partially a remnant of that, and isn't needed.